app/javascript/client/home/HomePage.tsx

```diff
@@ -71,11 +72,42 @@ interface HomePageProps {
     products: ProductInfoFragment[];
 }
 
+interface County {
+    id: string,
+    name: string
+}
+
+interface Category {
+    id: string,
+    name: string
+}
+
 export const HomePage: FC<HomePageProps> = ({ products }) => {
-    
-    const ProductThumbnails = Object.entries(products).map((product)=>(
-        <ProductThumbnail key={product[0]} product={product[1]} />
-    ))
+
+    const {countyFilter, categoryFilter} = useContext(HomepageContext);
+
+    const belongsToCounty = (county: County) => {
+        return county.name == countyFilter;
+    };
+
+    const belongsToCategory = (category: Category) => {
+        return category.name == categoryFilter;
+    };
+
+    const PriorityProductThumbnails = Object.entries(products).map((product)=>{
```

Comment:
Priority products are the signs that actually have counties/categories associated with them

Reply:
Sweet, thanks for explaining. Maybe in a future PR you could add that very explanation above this code as a comment ??